hack/s3_test/test1/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM public.ecr.aws/debian/debian:bullseye-slim
+
+RUN cat /dev/urandom | head -c 100 | sha256sum > unique_first
+RUN cat /dev/urandom | head -c 100 | sha256sum > unique_second
```

Comment:
Add a another stage in here

```
FROM scratch
COPY --link --from=build /unique_first /
COPY --link --from=build /unique_second /
```

And in the test add `--output destdir` . After cache import the contents of the files need to remain the same.

Reply:
Done